ruby/ql/test/library-tests/ast/constants/constants.rb

```diff
@@ -40,3 +40,22 @@ module ModuleA::ModuleC
 
 puts GREETING
 puts ::GREETING
+
+module ModuleA::ModuleB
+  class ClassB < Base
+  end
```

Comment:
Let's add a few constant definitions to the classes and modules defined in the examples.

```
module ModuleA::ModuleB
  class ClassB < Base
    X = 42
  end
end
``` 

Similar for the others.